src/parser/pegen_types.ts

```diff
@@ -93,3 +93,8 @@ export const FUNC_TYPE_INPUT = 345;
 export const FSTRING_INPUT = 800;
 
 export type StartRule = 256 | 257 | 258 | 345 | 800;
+
+export function EXTRA_EXPR(head: expr, tail?: expr): Attrs {
+    tail ??= head;
```

Comment:
what's this magic?

Reply:
if tail is null set it to the head
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment

Reply:
I can also just change this to make the second argument required like the cpython macro does 🤷‍♂️

Reply:
ohh cool